less/navigation.less

```diff
@@ -1 +1,33 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.navsec {
```

Comment:
remember to add () after the mixin name so that you don't accidently use it as a class. It won't really make a difference here in this small project, but once you have 5k+ lines of code, you may accidentally use it.

Also, please move this over to your mixins file. That's what its there for :)